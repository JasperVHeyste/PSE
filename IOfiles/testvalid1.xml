<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<DEVICE>
		<name>Office_Printer5</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>12</cost>
		<type>bw</type>
	</DEVICE>
	<JOB>
		<jobNumber>89751</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
		<type>scan</type>
	</JOB>
	<JOB>
		<jobNumber>2189</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
	</JOB>
	<DEVICE>
		<name>Office_Printer6</name>
		<emissions>2</emissions>
		<speed>30</speed>
		<type>color</type>
	</DEVICE>
</SYSTEM>
