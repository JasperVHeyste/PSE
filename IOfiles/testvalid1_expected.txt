device
Office_Printer5
3
40
bw
12

job
89751
2
SergeDemeyer
scan

job
2189
3
anonymous_user

device
Office_Printer6
2
30
color

